wiki/Medals/en.md

```diff
@@ -1,59 +1,295 @@
+| Medal name | Requirement(s) |
+| :-- | :-- |
+| Finality | Clear any Qualified, Ranked, or Approved beatmap with the [SuddenDeath](/wiki/Game_modifier/Sudden_Death) mod enabled. |
+| Perfectionist | Clear any Qualified, Ranked, or Approved beatmap with the [Perfect](wiki/Game_modifier/Perfect) mod enabled. |
+| Rock Around The Clock | Clear any Qualified, Ranked, or Approved beatmap with the [HardRock](/wiki/Game_modifier/Hard_Rock) mod enabled. |
+| Time And A Half | Clear any Qualified, Ranked, or Approved beatmap with the [DoubleTime](/wiki/Game_modifier/Double_Time) mod enabled. |
+| Sweet Rave Party | Clear any Qualified, Ranked, or Approved beatmap with the [Nightcore](/wiki/Game_modifier/Nightcore) mod enabled. |
+| Blindsight | Clear any Qualified, Ranked, or Approved beatmap with the [Hidden](/wiki/Game_modifier/Hidden) mod enabled. |
+| Are You Afraid Of The Dark? | Clear any Qualified, Ranked, or Approved beatmap with the [Flashlight](/wiki/Game_modifier/Flashlight) mod enabled. |
+| Dial It Right Back | Clear any Qualified, Ranked, or Approved beatmap with the [Easy](/wiki/Game_modifier/Easy) mod enabled. |
+| Risk Averse | Clear any Qualified, Ranked, or Approved beatmap with the [NoFail](/wiki/Game_modifier/No_Fail) mod enabled. |
+| Slowboat | Clear any Qualified, Ranked, or Approved beatmap with the [HalfTime](/wiki/Game_modifier/Half_Time) mod enabled. |
+| Burned Out | Clear any Qualified, Ranked, or Approved beatmap with the [SpunOut](/wiki/Game_modifier/Spun_Out) mod enabled. This medal can only be achieved in the osu! game mode as the SpunOut mod is not present in other game modes. |
 
 ## Hush-Hush
 
-Medals of this kind have no obvious way to get them. The player must find out what needs to be done in order to unlock the medal. The title itself is a clue.
+**Hush-Hush medals** are "secret" medals awarded for pulling off very specific feats, which are not explicitly told to the player (hence the name). These feats can range anywhere from completing a beatmap under peculiar circumstances to setting a score with an unusual mod combination. Despite their secretive nature, there are hints laying around to help all the curious medal hunters in their sleuth as follows:
+
+- The names of the medals themselves,
+- The medals' brief descriptions, and
+- The *flair texts* <!-- https://twitter.com/ephemeralis/status/1068054191602638850 --> of the medals (if any), viewable by hovering over their corresponding cards.
+
+As the Hush-Hush medals are meant to be secret achievements of some sort, the list below only note out the *descriptions and flair texts* of the medals instead of the exact method of getting them; however, players are [welcomed to work together in order to figure out how to obtain all the Hush-Hush medals out there](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```

Comment:
```suggestion
As the Hush-Hush medals are meant to be secret achievements of some sort, the list below only note out the *descriptions and flair texts* of the medals instead of the exact method of getting them; however, players are [welcome to work together in order to figure out how to obtain all the Hush-Hush medals out there](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).
```